From b06892c3da744abb1f5147ba8208ef0d7303416c Mon Sep 17 00:00:00 2001
From: Markus Armbruster <armbru@redhat.com>
Date: Wed, 31 Mar 2010 13:25:16 -0300
Subject: [PATCH 61/66] error: Trim includes after "Move qemu_error & friends..."

RH-Author: Markus Armbruster <armbru@redhat.com>
Message-id: <1270041921-28969-62-git-send-email-armbru@redhat.com>
Patchwork-id: 8228
O-Subject: [PATCH 61/66] error: Trim includes after "Move qemu_error &
	friends..."
Bugzilla: 579470
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>
RH-Acked-by: Luiz Capitulino <lcapitulino@redhat.com>

Missed in commit 2f792016.

Signed-off-by: Markus Armbruster <armbru@redhat.com>
Signed-off-by: Aurelien Jarno <aurelien@aurel32.net>
(cherry picked from commit 9f59b566a6569be148df0f2eb912b11c9d030deb)
---
 hw/qdev-properties.c |    1 +
 monitor.c            |    2 --
 sysemu.h             |    2 --
 3 files changed, 1 insertions(+), 4 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 hw/qdev-properties.c |    1 +
 monitor.c            |    2 --
 sysemu.h             |    2 --
 3 files changed, 1 insertions(+), 4 deletions(-)

diff --git a/hw/qdev-properties.c b/hw/qdev-properties.c
index 92d6793..157a111 100644
--- a/hw/qdev-properties.c
+++ b/hw/qdev-properties.c
@@ -1,6 +1,7 @@
 #include "sysemu.h"
 #include "net.h"
 #include "qdev.h"
+#include "qerror.h"
 
 void *qdev_get_prop_ptr(DeviceState *dev, Property *prop)
 {
diff --git a/monitor.c b/monitor.c
index fa21fe0..82b0901 100644
--- a/monitor.c
+++ b/monitor.c
@@ -50,10 +50,8 @@
 #include "qint.h"
 #include "qfloat.h"
 #include "qlist.h"
-#include "qdict.h"
 #include "qbool.h"
 #include "qstring.h"
-#include "qerror.h"
 #include "qjson.h"
 #include "json-streamer.h"
 #include "json-parser.h"
diff --git a/sysemu.h b/sysemu.h
index b1112bf..3f3e42e 100644
--- a/sysemu.h
+++ b/sysemu.h
@@ -6,8 +6,6 @@
 #include "qemu-option.h"
 #include "qemu-queue.h"
 #include "qemu-timer.h"
-#include "qdict.h"
-#include "qerror.h"
 
 #ifdef _WIN32
 #include <windows.h>
-- 
1.7.0.3

